source/server/getting-started/configuration/index.rst

```diff
@@ -2,81 +2,104 @@
 Configuring Sponge
 ==================
 
-You can find all configuration files inside the "config" folder.
-
-Config Syntax
-=============
-
-Most configuration files will make use of the :doc:`HOCON <hocon>` format.
-
 .. toctree::
     :maxdepth: 3
     :titlesonly:
 
+    sponge-conf
     hocon
-    json
 
-What You Can Configure
-======================
+.. _configuration_sponge:
 
-.. toctree::
-    :maxdepth: 3
-    :titlesonly:
+This article is about configuring Sponge itself, plugins will usually create their own configuration files inside
+the servers `config` directory.
+For Vanilla configuration check out the `Minecraft Wiki <https://minecraft.gamepedia.com/Server.properties>`__.
 
-    sponge-conf
-    server-properties
+Sponge Configuration Files
+==========================
+
+Sponge has multiple configuration files which all can be found in the ``config/sponge/`` directory of your server:
+
+* ``tracker.conf``
+* ``custom_data.conf``
+* ``global.conf``
+  and its dimension and world subconfigs in the ``worlds`` folder
 
-Plugins will also have their own configuration files in the "config" folder.
+The ``global.conf`` file contains the global configuration settings for Sponge. Its properties can be also overridden
+for each world and dimension type by using the config files in the subfolders of ``config/worlds``.
 
-World Configs
-=============
+The ``tracker.conf`` can be used to configure the Phase Tracker and the ``custom_data.conf`` is used to manage custom
+data added by plugins.
 
-There are three types of world configs:
+Modifying the Configuration Using a Text Editor
+===============================================
 
-* Global
-* Dimension
-* World
+Open the desired file using your preferred editor, the Sponge configuration files use HOCON, you can read more
+about that format in :doc:`"Introduction to HOCON" <hocon>`.
 
-Global configuration files can affect all of a server's worlds and dimensions. This is the default level for configs.
-Dimension configuration files are used to affect a certain dimension or group of worlds. These types of configs will
-override the global config files. World configuration files are used to modify individual worlds only. World configs
-override dimension and global configs.
+After making your desired changes, save your file. If your server was running while you made changes you have to reload
```

Comment:
A comma after "changes" would be nice. I'd change "have to" into "must", as it's shorter and more imperative.